cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java

```diff
@@ -1343,8 +1337,12 @@ public void testWorkflowForkFailure() throws Exception {
                                             WorkflowFailureInForkApp.SECOND_MAPREDUCE_NAME);
 
     String outputPath = new File(tmpFolder.newFolder(), "output").getAbsolutePath();
+    File fileToSync = new File(tmpFolder.newFolder() + "/sync.file");
```

Comment:
`tmpFolder.newFile("sync.file")` creates a new file right? here we only need a file instance without creating a file. Files will be created in MapReduce programs.